src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,114 @@
+
+- `fromHash` (optional): String containing a hexadecimal-encoded hash of the header of the block from which the storage difference will be calculated. When this parameter is provided, the storage difference is calculated between the `currentHash` block and the `fromHash` block. If this parameter is not provided, the storage difference is calculated between the `currentHash` block and the parent of the `currentHash` block.
+
+- `items` (optional): JSON object containing the following fields:
+
+    ```json
+    "items": {
+        "prefixes": [
+            {
+                "key": "0x...",
+                "type": "value" | "hash" | "none",
+            },
+        ],
+        "excludeKeyPrefixes": [
+            "0x...",
+        ],
+        "childTrie": "0x...",
+    }
+    ```
+
+  If the `items` parameter is not provided, the storage difference is calculated for all keys of the main storage trie.
+  - `prefixes` (optional): Array of JSON objects describing how the storage difference will be calculated. Each object contains the following fields:
+    - `key`: String containing the hexadecimal-encoded key prefix under which the storage difference is calculated.
+    - `type`: String equal to one of: `value`, `hash`, `none`.
+  - containing the key prefixes for which the storage difference will be calculated. If this parameter is not provided, the storage difference is calculated for all keys.
+  - `excludeKeyPrefixes` (optional): Array of strings containing the key prefixes for which the storage difference will not be calculated. If this parameter is not provided, the storage difference is calculated for all keys.
+  - `childTrie` (optional): A string containing the hexadecimal-encoded key of the child trie of the "default" namespace. If this parameter is not provided, the storage difference is calculated for the main storage.
+
+**Return value**: String containing an opaque value representing the operation.
+
+This function calculates the storage difference between two blocks. The storage difference is calculated by comparing the storage of the `fromHash` block with the storage of the `currentHash` block. If the `fromHash` parameter is not provided, the storage difference is calculated between the parent of the `currentHash` block and the `currentHash` block.
+
+The storage difference is calculated for the main storage trie by default. The `items` parameter can be used to specify the key prefixes for which the storage difference will be calculated. The `prefixes` field contains an array of objects, each describing a key prefix and the type of the storage difference to calculate. The `excludeKeyPrefixes` field contains an array of key prefixes for which the storage difference will not be calculated. The `childTrie` field specifies the child trie for which the storage difference will be calculated.
```

Comment:
THese bits feel a bit duplicated, ie after reading the parameters stuff I already know these things. Perhaps it's worth a bit of duplication but unsure how much to keep!